src/meta/src/barrier/command.rs

```diff
@@ -1074,6 +1195,59 @@ impl CommandContext {
                     .await;
             }
 
+            Command::CreateSnapshotBackfillStreamingJob { info, .. } => {
```

Comment:
Why does Backfill Streaming job need its own command? Why can't it just reuse that of normal streaming jobs

Reply:
Previously introduce a new command so that I can be aware of how the new snapshot backfill will affect the current logic. But it seems to be more elegant to reuse the original CreateStreamingJob command, so I change to reuse it in the latest commit.

I will update the PR description soon for easier review.